api/config/cdr_config_preprod.json

```
@@ -110,12 +110,12 @@
       "name": "All of Us Controlled Tier Dataset v5",
       "isDefault": true,
       "bigqueryProject": "fc-aou-cdr-preprod-ct",
-      "bigqueryDataset": "C2021Q3R2",
+      "bigqueryDataset": "C2021Q3R5",
       "wgsBigqueryDataset": "beta_release",
```

This will make it consistent with the WGS prep table you used

```suggestion
      "wgsBigqueryDataset": "alpha2_prelim2",
```